### ISSUE TITLE
import CREDIT ontology to CRO

### ISSUE THREAD

**Issue by nicolevasilevsky** [data2health/architecting_attribution#9]
import the whole CREDIT ontology as an owl:imports (on the ontology metadata page in Protege)

**Comment by marijane** [comment]
This is dependent on fixing the w3id for the CREDIT ontology -- Protege won't import an ontology from a url that doesn't end in .owl, apparently.

**Comment by marijane** [comment]
Progress!  I have pushed a refactored cro-edit.owl:https://github.com/data2health/contributor-role-ontology/commit/c4daca3eccfb6332871af24031d66a840e266787

I need to fix one more thing in the credit-ontology and do another release of that, then I can merge the refactor in and do another CRO release.

One thing I wasn't sure what to do about was the fact that each ontology has a root-level "contributor role" class.  I initially considered deprecating the one in CRO, but then I changed my mind and made it equivalent to the one in CREDIT.
